### PR TITLE
Added counter input fields

### PR DESCRIPTION
**Pull request by visualapps**
Screen Shot 2020-02-25 at 8 44 40 PM

### CODE REVIEW

File: scss/product/components/_forms.scss [hunk 1]
```diff
@@ -197,87 +180,97 @@ select.form-control-lg {
...
-  > .btn:not(:first-child),
-  > .btn-group:not(:first-child) > .btn {
+  .form-control {
+    box-shadow: none;
+    text-align: center;
+    @include border-right-radius(0);
     @include border-left-radius(0);
   }
+
+  .counter-input-number-field { }
```
Comment (Pallavi-PH) [inline scss/product/components/_forms.scss]: Since no CSS is added for this class, this line `.counter-input-number-field { }` can be removed

**Comment by Pallavi-PH** [comment]
I think the unit value `MB` should be center-aligned for Medium size counter input, similar to Large size counter input
Screenshot from 2020-02-27 12-13-59

**Comment by visualapps** [comment]
> I think the unit value `MB` should be center-aligned for Medium size counter input, similar to Large size counter input
> Screenshot from 2020-02-27 12-13-59

updated the CSS for center align

File: scss/product/components/_forms.scss (hunk 1 above)
Comment (visualapps) [reply scss/product/components/_forms.scss]: removed

**Pull request merged**